### PR TITLE
Feature create data modal

### PR DESCRIPTION
**Pull request by vikipretium**
## Project requirements
Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.

- Build your project schema. ✔️ 
- Create and run the necessary migration files. ✔️
- Table and column names should match the ERD diagram. ✔️
note: photo for the users table should be a link to an image
- Foreign keys should be included. ✔️
- All columns that are foreign keys should have a corresponding index. ✔️

### CODE REVIEW

File: db/migrate/20220611174954_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.belongs_to :users, null: false, foreign_key: true
```
Comment (medaminedev66) [inline db/migrate/20220611174954_create_posts.rb]: - [x] The reference is not directly `user`, it is instead `author` to table `user`. Kindly see the below example on how to make such migration for adding reference with a different name than the reference model class name and make changes in your database. :pray: 

image

File: db/migrate/20220611175533_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.belongs_to :users, null: false, foreign_key: true
```
Comment (medaminedev66) [inline db/migrate/20220611175533_create_comments.rb]: - [x] Again, the reference name should be `author`. Please refer to the above similar comment for the other migration. :pray:

File: db/migrate/20220611175705_create_likes.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.belongs_to :users, null: false, foreign_key: true
```
Comment (medaminedev66) [inline db/migrate/20220611175705_create_likes.rb]: - [x] Again, the reference name should be `author`. Please refer to the above similar comment for the other migration. :pray:

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @vikipretium  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- `rails db:migrate` command runs successfully.
- There are no linter errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220611174954_create_posts.rb (hunk 1 above)
Comment (vikipretium) [reply db/migrate/20220611174954_create_posts.rb]: Sure @medaminedev66 
I'll fix it 👍🏼

File: db/migrate/20220611175533_create_comments.rb (hunk 2 above)
Comment (vikipretium) [reply db/migrate/20220611175533_create_comments.rb]: I fixed it @medaminedev66

File: db/migrate/20220611175705_create_likes.rb (hunk 3 above)
Comment (vikipretium) [reply db/migrate/20220611175705_create_likes.rb]: I fixed it @medaminedev66 👍🏼

**Review by afanimartin — APPROVED** [review]
Hi  @vikipretium, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**